configs/ecospheres/config.yaml

```diff
@@ -52,6 +52,61 @@ website:
   show_topic_charts: false
   list_highlighted_topics:
   oauth_option: true
+  selectedThemeDefault: 'Produire'
```

Comment:
La config gagnerait à être simplifiée. les répétitions vont rendre le truc compliqué à maintenir.

Je propose quelque chose comme : 
```
themes:
  - name: Produire
    default: true
    subthemes:
      - name: Nucléaire
        default: true
      - name: Tertiaire (incluant les bâtiments publics de l'État et des collectivités, hors logement)
      - name: Production d'électricité décarbonée (hors nucléaire)
      ...
  - name: Se nourrir
    subthemes:
      - name: Alimentation
        default: true
      - Agriculture et pêche
    ...
```

Reply:
Discuté en hebdo : 
Pas besoin de valeur par défaut car on veut forcer l'utilisateur à choisir le thème et chantier du bouquet explicitement. Les deux dropdown doivent être vides par défaut et le bouton "suivant" désactivé. A chaque changement de sélection, le chantier doit être réinitialisé à vide. 

Côté conf, ça veut dire qu'on peut simplifier encore plus et virer `default`. Deux options : 

_Option 1: On vire juste la clé `default`_
```
themes:
  - name: Produire
    subthemes:
      - name: Nucléaire
      - name: Tertiaire (incluant les bâtiments publics de l'État et des collectivités, hors logement)
      - name: Production d'électricité décarbonée (hors nucléaire)
      ...
  - name: Se nourrir
    subthemes:
      - name: Alimentation
      - Agriculture et pêche
    ...
```

_Option 2: On vire carrément les clés_
```
themes:
  - Produire: 
    - Nucléaire
    - Tertiaire (incluant les bâtiments publics de l'État et des collectivités, hors logement)
    - Production d'électricité décarbonée (hors nucléaire)
    ...
  - Se nourrir:
    - Alimentation
    - Agriculture et pêche
    ...
```

L'option 2 est plus simple visuellement mais plus embêtant à manipuler côté js et moins évolutif si jamais on veut rajouter des propriétés à l'avenir.

Donc j'aurais tendance à voter pour l'option 1.

Reply:
+1 pour l'option 1

Reply:
Timeout pour les contributions ;) Go pour option 1.
FYI @YeLnatSs

Reply:
I made the changes for the option 1